### PR TITLE
Updated the artifact_version to use an ARG isntead of an ENV to prevent…

### PR DESCRIPTION
**Pull request by zmoore-usgs**
… it from overriding the value set by downstream images if they use an ARG instead of an ENV (which is what we should be using anyways).

### CODE REVIEW

File: 11/Dockerfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ ARG DEFAULT_JAVA_OPTIONS="-Xmx300M -server"
 
 LABEL maintainer="gs-w_eto_eb_federal_employees@usgs.gov"
 
-ENV artifact_version=0.0.1-SNAPSHOT
+ARG artifact_version=set_the_build_number_in_your_dockerfile
```
Comment (isuftin) [inline 11/Dockerfile]: This string doesn't have quotes (should it?) and is different from the others

Comment (zmoore-usgs) [reply 11/Dockerfile]: Whoops, yeah this was the first version and then I updated it and forgot to copy it back. Will fix.

Comment (zmoore-usgs) [reply 11/Dockerfile]: Actually I did copy it back and forgot to hit CTRL+S apparently. 🤦‍♂️

**Comment by isuftin** [comment]
Once the string is quoted and updated to match, you should be all set

**Comment by cschroed-usgs** [comment]
Sorry, I don't understand the interactions with the downstream images. Could you please link me to a downstream Dockerfile that illustrates the issue?

Also, why does the base image define `artifact_version`? It doesn't appear to be used.

**Comment by zmoore-usgs** [comment]
The issue was with the AQCU Java to R image. I sent you a link to the build on Slack.

The base image defines it to be an example of how downstream images should define it. It's used in downstream images for the pull from Artifactory script.

**Comment by zmoore-usgs** [comment]
I can add a note to the readme

**Pull request merged**